.changeset/violet-parents-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+'@remote-ui/async-subscription': patch
+---
+
+return subscriber result
```

Comment:
Can you add a bit more detail about how a consumer might use this? You can use a simplified version of what you will do with it in our production apps.